versions/draft-3/SPEC.md

```diff
@@ -1018,12 +1018,31 @@ task memory_test {
 
 ```
 $parameter_meta = 'parameter_meta' $ws* '{' ($ws* $parameter_meta_kv $ws*)* '}'
-$parameter_meta_kv = $identifier $ws* '=' $ws* $string
+$parameter_meta_kv = $identifier $ws* ':' $ws* $expression
 ```
 
-This purely optional section contains key/value pairs where the keys are names of parameters and the values are string descriptions for those parameters.
+This purely optional section contains key/value pairs where the keys are names of parameters and the values are literal expressions
+that describe those parameters. Note that the expressions *cannot* reference variables. The engine can ignore this section, with no loss of correctness. The extra information can be used, for example, to generate a user interface.
```

Comment:
IMO a new grammar element which enforces this is better than parsing things like `3+ mytask.out` and then (failing the wf? ignoring it?).

It sounds like a copy/paste of the `object` literal with functions and lookups removed.